pages/index.js

```diff
@@ -1,7 +1,31 @@
+import { initialProjects } from "@/db/data";
+import Project from "@/components/Project";
+import styled from "styled-components";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <StyledHeadline>DIY APP</StyledHeadline>
+      <StyledSection>
+        {initialProjects.map((initialProject) => (
+          <Project
+            key={initialProject.id}
+            title={initialProject.title}
+            duration={initialProject.duration}
+            complexity={initialProject.complexity}
+          />
+        ))}
+      </StyledSection>
+    </>
   );
 }
+
+const StyledSection = styled.section`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+`;
```

Comment:
And for the parent StyledSection, you can define the `gap` property like so.  This will create a 20px gap between the articles, for example.

```suggestion
const StyledSection = styled.section`
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  gap: 20px;
`;
```

Reply:
Done